src/commands/version.js

```diff
@@ -0,0 +1,3 @@
+export default function version() {
+    console.log('SernHandler CLI v0.1.1');
```

Comment:
use package.json to get version instead of hard coding it

Reply:
I’m mazy for that

Reply:
no